src/stories/Library/opening-hours/OpeningHoursExampleData.ts

```diff
@@ -0,0 +1,250 @@
+// These types are based on the actual data structure returned from the API.
+// The types starting with DplOpeningHours.. are generated with Orval in react
+// and copy pasted here.
+export type DplOpeningHoursListGET200Item = {
```

Comment:
:+1: Appreciate the commenting here.

Normally I would not think that these "technicalities" belonged in the design system"